common/persistence/sql/dbal/class.Driver.php

```diff
@@ -52,12 +58,60 @@ function connect($id, array $params)
         $config = new \Doctrine\DBAL\Configuration();
 //          $logger = new Doctrine\DBAL\Logging\EchoSQLLogger();
```

Comment:
Do we have a good reason to leave this commented code in?

Reply:
someone left it here, I guess that it can simplify debugging

Reply:
That's what I thought, just wanted to check ;)